app/uk/gov/hmrc/partnershipidentificationfrontend/service/ValidationOrchestrationService.scala

```diff
@@ -29,15 +29,15 @@ class ValidationOrchestrationService @Inject()(partnershipIdentificationService:
 
   def orchestrate(journeyId: String, businessVerificationCheck: Boolean)(implicit hc: HeaderCarrier): Future[ValidationResponse] =
     partnershipIdentificationService.retrievePartnershipInformation(journeyId).flatMap {
-      case Some(PartnershipInformation(Some(SaInformation(sautr, postcode)), Some(_))) =>
+      case Some(PartnershipInformation(Some(SaInformation(sautr, postcode)), Some(_: CompanyProfile))) =>
```

Comment:
Does this type check need to be here? I can't see that it's being used, and it will always be CompanyProfile.

Reply:
You are right .... I put just because from that expression .. Some(_)I dont know the type of  Option. I think is more readable but I can remove it without a problem